sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/experiment_events_live_v1/init.sql

```diff
@@ -20,21 +20,18 @@ IF
     SELECT
       submission_timestamp AS `timestamp`,
       event.category AS `type`,
-    -- using UNNEST twice on the same field doesn't work in materialized views
-      REGEXP_EXTRACT(
-        TO_JSON_STRING(event.extra),
-        '.*{"key":"branch","value":"([^"]*)"}.*'
-      ) AS branch,
-      REGEXP_EXTRACT(
-        TO_JSON_STRING(event.extra),
-        '.*{"key":"experiment","value":"([^"]*)"}.*'
-      ) AS experiment,
+      CAST(event.extra[safe_offset(i)].value AS STRING) AS branch,
+      CAST(event.extra[safe_offset(j)].value AS STRING) AS experiment,
       event.name AS event_method
     FROM
       fenix_all_events,
-      UNNEST(events) AS event
+      UNNEST(events) AS event,
+      UNNEST(GENERATE_ARRAY(0, 50)) AS i,
```

Comment:
Can you add a comment here to mention that this is a work-around for known incorrect behavior and point to the public bug or the GCP caes?